im_library/src/interfaces/PropertyShape.ts

```diff
@@ -1,4 +1,4 @@
-import { Argument } from "./query/Argument";
+import { Argument } from "../models/AutoGen";
```

Comment:
I was having issues with the editor autogen interfaces. Did you check this change worked in the editor/creator?